rpcs/btc/BtcRpc.js

```diff
@@ -47,6 +47,20 @@ class BtcRpc {
     return tx;
   }
 
+  async unlockAndSendToAddressMany({ payToArray, passphrase }) {
+    if (passphrase === undefined) {
+      passphrase = await promptly.password('> ');
+    }
+    await this.asyncCall('walletPassPhrase', [passphrase, 10]);
+    const promises = payToArray.map(a => {
+      const [address, amount] = a;
+      return this.sendToAddress({ address, amount });
+    });
+    const txs = await Promise.all(promises);
```

Comment:
I don't think that Promise.all gives us the behavior that we want... if one of the transactions fails then it would reject without giving us information about any that succeeded